pkg/subctl/cmd/join.go

```diff
@@ -127,7 +127,8 @@ func addJoinFlags(cmd *cobra.Command) {
 		"interval in seconds between health check packets")
 	cmd.Flags().Uint64Var(&healthCheckMaxPacketLossCount, "health-check-max-packet-loss-count", 5,
 		"maximum number of packets lost before the connection is marked as down")
-	cmd.Flags().BoolVar(&allocGNet, "allocate-globalnet", true, "Allocate Globalnet CIDR")
+	cmd.Flags().BoolVar(&globalnetEnabled, "globalnet-enabled", true,
```

Comment:
I think @skitt was trying to move away from "enabled" in flags. Maybe just `globalnet`?

Reply:
ahh - OK.  I think just `globalnet` makes sense.